src/utility/validator.utility.ts

```diff
@@ -0,0 +1,9 @@
+class Validator {
+  public constructor() {}
+
+  static isNotNullOrUndefined(value: any[]): boolean {
+    return !value.every((val) => val !== null && val !== undefined);
+  }
```

Comment:
I like this. Don't forget to be explicit about visibility, though. Readability is important.